docs/semgrep-assistant/customize.md

```diff
@@ -47,7 +47,7 @@ Findings filtered out by Assistant can be reviewed at any time in Semgrep by goi
 
 ## Add Memories
 
-Assistant Memories allows admins to tailor Assistant's remediation guidance to their organization's standards and defaults on a per-project, per-rule basis. You can provide feedback by adding custom instructions whenever Assistant gives a suggested fix.
+Assistant Memories allow admins to tailor Assistant's remediation guidance to their organization's standards and defaults. You can provide feedback by adding custom instructions whenever Assistant gives a suggested fix.
```

Comment:
allow sounds less weird to me, but if it's supposed to be understood as a singular feature, I can revert that. For the other change here, it seemed clunky to add `per vulnerability class` so I opted to go more general.